c2corg_ui/templates/index.html

```diff
@@ -38,4 +38,20 @@ <h1 class="welcome-title text-center" translate>The mountain sports community</h
     </div>
   </section>
   <app-feed id="feed"></app-feed>
+
+  <div class="float-buttons" ng-if="userCtrl.auth.isAuthenticated()">
+    <div class="float-button-container">
+      <div class="float-button" ng-click="feedCtrl.turnPersonalFeedOnOff()" ng-class="{'personal-feed-on': feedCtrl.personalFeed}">
+        <span class="glyphicon glyphicon-user"></span>
+      </div>
+      <p class="float-button-text" x-translate ng-bind="feedCtrl.personalFeed ? 'Personal feed on' : 'Personal feed off'">Personal feed</p>
```

Comment:
We need to find a way to make ``Personal feed on`` and ``Personal feed off`` be extracted by the i18n tool.
Translate filters maybe?
At least add them to the ``i18n.html`` template but that's a poor solution.